ocaml/xapi/workload_balancing.ml

```diff
@@ -24,6 +24,16 @@ open D
 
 exception Xml_parse_failure of string
 
+(* This is a stub used in the pattern_matching below to silence a
+ * warning in the newer ocaml compilers *)
+exception Float_of_string_failure
```

Comment:
Maybe add the string that we can't concert to this error such that we can report it.

Reply:
The string of the Failure is simply `"float_of_string"`, unfortunately. We're not removing any information doing this.